src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Symbols/MethodOrPropertySymbol.cs

```diff
@@ -29,7 +29,6 @@ internal abstract class MethodOrPropertySymbol : ParentSymbol
         public new bool isStatic;               // Static member?
         public bool isOverride;             // Overrides an inherited member. Only valid if isVirtual is set.
         // false implies that a new vtable slot is required for this method.
-        public bool useMethInstead;         // Only valid iff isBogus == TRUE && IsPropertySymbol().
         // If this is true then tell the user to call the accessors directly.
```

Comment:
This comment refers to the code that is now gone. Also the comment above looks like after the code it is referring to. Can that be fixed too?